.azure-pipelines/scripts/Get-SigntoolPath.ps1

```diff
@@ -0,0 +1,26 @@
+function Get-Signtool() {
+  <#
+    .SYNOPSIS
+    Function used to get signtool from windows SDK
+  #>
+
+  [System.IntPtr]::Size # an integer with platform-specific size.
+  $systemBit = "x64"
+  $programFiles = ${Env:ProgramFiles(x86)}
+
+  if([System.IntPtr]::Size -eq 4) {
+    $systemBit = "x86"
+    $programFiles = ${Env:ProgramFiles}
+  }
+
+  $signtoolPath = ""
+  try {
+    $windowsSdkPath=Get-ChildItem "$programFiles\Windows Kits\10\bin\1*" | Select-Object FullName | Sort-Object -Descending { [version](Split-Path $_.FullName -leaf)} | Select-Object -first 1
+
+    $signtoolPath = "$($windowsSdkPath.FullName)\$systemBit\signtool.exe"
+    return $signtoolPath
+  } catch {
+    Write-Error "Unbable to get signtool in $signtoolPath"
```

Comment:
Considering the fact that this script will be executed in the pipeline. I think we can use [formatting commands](https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=powershell#formatting-commands) for errors, warnings, and debug information.

Reply:
Added formatting commands for errors, not sure that there are any info for which I could use warnings and debug commands